packages/zosjobs/__tests__/__system__/DownloadJobs.system.test.ts

```diff
@@ -79,6 +78,7 @@ describe("Download Jobs - System tests", () => {
 
     afterAll(async () => {
         await DeleteJobs.deleteJob(REAL_SESSION, jobname, jobid);
+        await TestEnvironment.cleanUp(testEnvironment);
```

Comment:
Should we remove this `DeleteJobs.deleteJob` call from here (since we added the job as part of the terEnv resources? 😋